src/Server.ts

```diff
@@ -0,0 +1,25 @@
+import * as dotenv from 'dotenv';
+dotenv.config();
+
+import express from 'express';
+import path from 'path';
+
+import { Logger } from './logger/Logger';
+import { userRoute } from './routes/UserRoutes';
+import { authenticationRoute } from './routes/AuthenticationRoutes';
+
+const server = express();
+
+server.listen(5000, () => {
+    console.log('🚀 Server is running');
+});
+
+server.use(Logger.consoleLog);
+server.use(express.static(path.resolve(__dirname, './api/view/html/public')));
```

Comment:
I'm not familiar with express/node yet, but this seems strange - shouldn't the view be separate from the API?

Reply:
This is not view. This is middleware. In this case we define that all static html pages will be stored in a public folder. Since it's part of the routing it has to be where it is.

Reply:
Also, a folder named public because it's a standard.

Reply:
> I'm not familiar with express/node yet, but this seems strange - shouldn't the view be separate from the API?

When it comes to view. Express follows the MVC pattern. View is related to htmls and stuff.. like remember how we had html folder in LAMP? Same thing applies to this case. Our view is basically our web page.